apps/web/src/scenes/NftDetailPage/NftDetailText.tsx

```diff
@@ -130,24 +129,16 @@ function NftDetailText({ tokenRef, authenticatedUserOwnsAsset }: Props) {
   }, [isMobile, showModal, token]);
 
   const CreatorComponent = useMemo(() => {
-    if (token.owner && token.ownerIsCreator) {
-      return (
-        <UserHoverCard userRef={token.owner}>
-          <OwnerProfilePictureAndUsername
-            userRef={token.owner}
-            eventContext={contexts['NFT Detail']}
-          />
-        </UserHoverCard>
-      );
-    }
     if (token.community?.creator) {
       if (token.community.creator.__typename === 'GalleryUser') {
```

Comment:
same comment on consolidating logic, tossing CreatorComponent, doing in-line ternary